cms/djangoapps/contentstore/views/course.py

```diff
@@ -483,10 +483,20 @@ def filter_ccx(course_access):
     courses_list = []
     course_keys = {}
```

Comment:
nit: I don't think `course_keys` here should be a dictionary we are eventually transforming it into a list so it should be list instead of dictionary.